src/doom/d_main.cpp

```diff
@@ -1129,11 +1129,30 @@ void DoomLoop::Iter(gvizdoom::Context& context,
     out_gameState.set<gvizdoom::GameState::DamageCount>(player->damagecount);
     out_gameState.set<gvizdoom::GameState::BonusCount>(player->bonuscount);
     out_gameState.set<gvizdoom::GameState::OnGround>(player->onground);
-    out_gameState.set<gvizdoom::GameState::X>(pos.X);
-    out_gameState.set<gvizdoom::GameState::Y>(pos.Y);
-    out_gameState.set<gvizdoom::GameState::Z>(pos.Z);
-    out_gameState.set<gvizdoom::GameState::Angle>(pos.Angle().Degrees);
+    out_gameState.set<gvizdoom::GameState::PlayerPos>(gvizdoom::Vec3f(pos.X, pos.Y, pos.Z));
```

Comment:
nit: technically should be on different commit, the `Player` prefix is a good idea though